src/source/JavaGenerator.scala

```diff
@@ -89,6 +92,7 @@ class JavaGenerator(spec: Spec) extends Generator(spec) {
     for (c <- consts) {
       writeDoc(w, c.doc)
       javaAnnotationHeader.foreach(w.wl)
+      marshal.nullityAnnotation(c.ty).foreach(w.wl)
```

Comment:
This is on a constant field, not a class definition (or I'm very confused here). In fact, looking at this again, it kinda seems like maybe the previous line (the `javaAnnotationHeader.foreach(w.wl)` line) shouldn't actually be here?